### ISSUE TITLE
Development is limited to Windows only?

### ISSUE THREAD

**Issue by taboxdev** [mitosisX/Limer-Limekit#2]
I own Mac and  judging by  what is published under Releases folder, development with this framework is limited to windows only. Please confirm.

**Comment by mitosisX** [comment]
Yes. Considering that I have no Mac for testing, the release was only limited to Windows.

**Comment by taboxdev** [comment]
Is the output still cross platform?
